src/admin_extra/service_admin_extra_upload.erl

```diff
@@ -0,0 +1,54 @@
+-module(service_admin_extra_upload).
```

Comment:
As far as I can tell there are no tests for these new ctl commands. It'd be really good to have them added. There suite `big_tests/tests/ejabberdctl_SUITE.erl` where such tests could be added. They can also be added to the  `big_tests/tests/mod_http_upload_SUITE.erl`